pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -654,8 +653,8 @@ func TestVolumeBinding(t *testing.T) {
 
 			t.Logf("Verify: call PreFilter and check status")
 			_, gotPreFilterStatus := p.PreFilter(ctx, state, item.pod)
-			if !reflect.DeepEqual(gotPreFilterStatus, item.wantPreFilterStatus) {
-				t.Errorf("filter prefilter status does not match: %v, want: %v", gotPreFilterStatus, item.wantPreFilterStatus)
+			if diff := cmp.Diff(item.wantPreFilterStatus, gotPreFilterStatus); diff != "" {
+				t.Errorf("filter prefilter status does not match: (-want,+got):\n%s", diff)
```

Comment:
It seems to be a typo.

```suggestion
				t.Errorf("prefilter status does not match: (-want,+got):\n%s", diff)
```